shared/src/common/enums/api/episodes-api-path.enum.ts

```diff
@@ -1,5 +1,6 @@
 enum EpisodesApiPath {
   ROOT = '/',
+  ID = '/:id',
```

Comment:
```suggestion
  $ID = '/:id',
```
dynamic parametres should be with `$`